### PR TITLE
Added address info for modbus.binary_sensor

### PR DESCRIPTION
**Pull request by dontbug**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @dontbug, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17340).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by dontbug** [comment]
> Hi @dontbug,
> 
> It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant/home-assistant.io#17340).
> 
> Once you do that we will be able to review and accept this pull request.
> 
> Thanks!

done

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -188,6 +188,10 @@ binary_sensors:
       description: The number of the slave.
       required: false
       type: integer
+    address:
+      description: Address of the Register.
+      required: false
```
Comment (frenck) [inline source/_integrations/modbus.markdown]: This seems to be required?

```py

BINARY_SENSOR_SCHEMA = BASE_COMPONENT_SCHEMA.extend(
    {
        vol.Required(CONF_ADDRESS): cv.positive_int,
        vol.Optional(CONF_DEVICE_CLASS): BINARY_SENSOR_DEVICE_CLASSES_SCHEMA,
        vol.Optional(CONF_INPUT_TYPE, default=CALL_TYPE_COIL): vol.In(
            [CALL_TYPE_COIL, CALL_TYPE_DISCRETE]
        ),
    }
)
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (dontbug) [reply source/_integrations/modbus.markdown]: yes. Thanks )). sorry

**Review by frenck — APPROVED** [review]
Thanks, @dontbug 👍

**Pull request merged**